modules/dxp/apps/segments/segments-asah-connector/src/main/java/com/liferay/segments/asah/connector/internal/messaging/IndividualSegmentsChecker.java

```diff
@@ -59,16 +62,16 @@
 public class IndividualSegmentsChecker {
 
 	public void checkIndividualSegments() {
-		Optional<AsahFaroBackendClient> asahFaroBackendClientOptional =
-			_asahFaroBackendClientFactory.createAsahFaroBackendClient();
+		List<Company> companies = _companyLocalService.getCompanies();
```

Comment:
I think maybe is risky to get all the companies in one list, we should use a pagination search :).

Reply:
Good catch, not an ActionableDynamicQuery is used ;)